src/twisted/words/protocols/jabber/client.py

```diff
@@ -354,16 +358,24 @@ class XMPPAuthenticator(xmlstream.ConnectAuthenticator):
                resource binding step, and this is stored in this instance
                variable.
     @type jid: L{jid.JID}
+
     @ivar password: password to be used during SASL authentication.
     @type password: L{unicode}
+
+    @ivar contextFactory: An object which creates appropriately configured TLS
```

Comment:
As above, could this have a better name?  `contextFactory` is a reference to the deprecated interface for this.